nextstrain_profiles/100k/config.yaml

```diff
@@ -0,0 +1,33 @@
+
+S3_DST_BUCKET: "nextstrain-ncov-private/100k"
+S3_DST_ORIGINS: [needed-for-workflow-but-unused]
+deploy_url: needed_for_workflow_but_unused
+
+custom_rules:
+  - workflow/snakemake_rules/export_for_nextstrain.smk
+
+# Note: unaligned sequences are provided as "aligned" sequences to avoid an initial full-DB alignment
+inputs:
+  - name: gisaid
+    metadata: "s3://nextstrain-ncov-private/metadata.tsv.gz"
+    aligned: "s3://nextstrain-ncov-private/sequences.fasta.xz"
+    skip_sanitize_metadata: true
+
+builds:
+  100k:
+    subsampling_scheme: 100k_scheme
+
+# remove sequences without division label in US
+filter:
+  exclude_where: "division='USA'"
+
+subsampling:
+  100k_scheme:
+    50k_early:
+      group_by: "year month region"
+      max_sequences: 50000
+      max_date: "--max-date 1Y"
+    50k_late:
+      group_by: "year month region"
+      max_sequences: 50000
+      min_date: "--min-date 1Y"
```

Comment:
Does this produce clean boundaries? Or will a certain month be (partially) double sampled?

Reply:
I'll say that this sample isn't supposed to be truly representative for proper analyses -- it's just a way to test things locally and get an idea of what the output would look like.

Reply:
I think if we get subsampling right, the "100k" dataset should produce nearly indistinguishable output for ncov global 6m (for example) to using the entire 14M sequence database.

Reply:
This should produce clean boundaries since `1Y` translates to [an exact day](https://github.com/nextstrain/augur/blob/a2d7660dd1c2c61814ce9291efb9cc87c0fe2567/augur/dates.py#L55).

Both `--min-date` and `--max-date` are [inclusive](https://github.com/nextstrain/augur/blob/a2d7660dd1c2c61814ce9291efb9cc87c0fe2567/augur/filter.py#L384-L388) so there may be double-sampling of that day, but no more than that.

Reply:
> This should produce clean boundaries since 1Y translates to [an exact day](https://github.com/nextstrain/augur/blob/a2d7660dd1c2c61814ce9291efb9cc87c0fe2567/augur/dates.py#L55).

Assuming the month that falls in is `X`, that will leave some samples from `X` in both samples (`50k_early`, `50k_late`). This'll partition things evenly according to month, so in `50k_early` there'll be `~50,000*1/24` from `X` and in `50k_late` there'll be `~50,000*1/12` samples. Won't matter for the 6M build, but a little less than ideal.

Reply:
Oh you're right, yes there will be double sampling of the month due to `--group-by month`.

The behavior with the exact date cutoff is worth pointing out. For example:

- `--max-date 2022-03-08 --group-by month` will sample the first week of March as much as each of the previous full months.
- `--min-date 2022-03-08 --group-by month` will sample the remainder of March as much as each of the following full months.

Reply:
Thanks for the clarifications! Added a comment to the build YAML explaining this.

P.S. This isn't just for our builds, it's going to happen everywhere. Even a simple single-sample scheme grouped by `year month` will oversample the current month unless we are running things on the final day of the month.